File: boyorgirl/boyorgirl.cpp
#include<iostream>
#include<string>
#include<vector>
#include<cstdlib>
#include<algorithm>

constexpr auto newline = "\n"; 
using std::string, std::vector; 
using std::cin, std::cout; 

int main()
{
    string uName; 
    cin >> uName; 
    int counter = 0; 
    string dist_l = ""; 

    for(auto it = uName.begin(); it < uName.end(); it++)
    {
        if (find(dist_l.begin(), dist_l.end(), *it) == dist_l.end())
        {
            dist_l.push_back(*it); 
        }
    }

    cout << ((dist_l.size() % 2 == 0) ? "CHAT WITH HER!" : "IGNORE HIM!") ;
} 

File: boyOrGirl/boyorgirl.py
uname = input()
i = 0
while(i < len(uname) - 1):

    if uname[i] in uname[i+1:]: 
        uname = uname[:i] + uname[i+1:]
        i -= 1
    i += 1
print("CHAT WITH HER!" if len(uname) % 2 == 0 else "IGNORE HIM!")

